sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/UnitTestResultsAggregatorTest.java

```diff
@@ -98,10 +99,9 @@ public void aggregate() {
     VisualStudioTestResultsFileParser visualStudioTestResultsFileParser = mock(VisualStudioTestResultsFileParser.class);
     NUnitTestResultsFileParser nunitTestResultsFileParser = mock(NUnitTestResultsFileParser.class);
     XUnitTestResultsFileParser xunitTestResultsFileParser = mock(XUnitTestResultsFileParser.class);
-    UnitTestResults results = mock(UnitTestResults.class);
     new UnitTestResultsAggregator(unitTestConf, settings.asConfig(), visualStudioTestResultsFileParser, nunitTestResultsFileParser, xunitTestResultsFileParser)
-      .aggregate(wildcardPatternFileProvider, results);
-    verify(visualStudioTestResultsFileParser).accept(new File("foo.trx"), results);
+      .aggregate(wildcardPatternFileProvider);
```

Comment:
I find this test really hard to read because it's doing Arrange-Act-Assert multiple times for VS Test , NUnit, XUnit and then some more.

For the clean as you code benefit, would it be worth splitting these in single tests that do only one set of Arrange-Act-Assert ?